vcf-dumper/vcf-dumper-lib/src/main/java/uk/ac/ebi/eva/vcfdump/VariantExporterController.java

```diff
@@ -383,16 +284,16 @@ public String getOutputFileName() {
         return outputFileName;
     }
 
-    public List<Region> divideChromosomeInChunks(String chromosome, int start, int end) {
+    public List<Region> divideChromosomeInChunks(String chromosome, long start, long end) {
         return regionFactory.divideChromosomeInChunks(chromosome, start, end);
     }
 
-    public int getCoordinateOfFirstVariant(String chromosome) {
-        return regionFactory.getMinStart(chromosome, query);
+    public long getCoordinateOfFirstVariant(String chromosome) {
+        return regionFactory.getMinStart(chromosome, query.getStudies());
```

Comment:
Call the service directly instead of the regionFactory method. The service is instantiated in this class, so we can call it directly.